Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Amazon.Lambda.Annotations.SourceGenerator.csproj

```diff
@@ -1,26 +1,50 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <AssemblyVersion>1.5.2</AssemblyVersion>
-    <TargetFramework>netstandard2.0</TargetFramework>
-
-    <!--This assembly needs to access internal methods inside the Amazon.Lambda.Annotations assembly.
+    <PackageId>Amazon.Lambda.Annotations</PackageId>
```

Comment:
To avoid having to rename the `Amazon.Lambda.Annotations` project can you remove this line and then add the following after the `PropertyGroup` it delays setting the `PackageId` till after projects are restored avoiding the ambiguous error. Then rename `Amazon.Lambda.Annotations.Models` back to its original name.

```
	<!-- Delay setting the package id till after resolving dependencies to avoid ambigious errors. -->
	<Target Name="SetPackageId" BeforeTargets="GenerateNuspec">
		<PropertyGroup>
			<PackageId>Amazon.Lambda.Annotations</PackageId>
		</PropertyGroup>
	</Target>
```

Reply:
Genius